src/views/Movies-Components/TopRatedMovies.js

```diff
@@ -0,0 +1,74 @@
+import { Grid } from "@mui/material";
+import React from "react";
+import { Navigation } from "swiper";
+import "swiper/css";
+import "swiper/css/navigation";
+import { Swiper, SwiperSlide } from "swiper/react";
+import Typography from "@mui/material/Typography";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import { CardActionArea } from "@mui/material";
+import { useGetTopRatedMoviesQuery } from "../../services/MovieServices";
+import { first_path } from "./Constants";
+export const TopRatedMovies = () => {
+  const { data } = useGetTopRatedMoviesQuery();
+
+  return (
+    <Grid item sx={{ gap: 2 }} width={500}>
+      <Typography
+        sx={{
+          fontSize: 16,
+          textAlign: "left",
+          color: "#046565",
+          fontWeight: "bold",
+          paddingBottom: 1,
+        }}
+      >
+        Top Rated
+      </Typography>
+      <Swiper navigation={true} modules={[Navigation]}>

```

Comment:
Now that I think, we could use a reusable component that has swiper. Wdyt?

Reply:
Okay